src/components/DashboardReports/DashboardReportsData.js

```diff
@@ -63,6 +64,14 @@ function DashboardReportsData() {
       iconColor: 'charcoal-600',
       auth: ASSISTANT,
     },
+    {
+      icon: 'CategoriesNone',
+      name: 'No categories',
```

Comment:
Minor: I am guessing non-programmers think of this as more like "Not in any categories" / "Words and phrases that haven't been categorized"